internal/instance/resource_instance.go

```diff
@@ -1336,13 +1336,23 @@ func (s sortedInterfaces) Swap(i, j int) {
 }
 
 func (s sortedInterfaces) Less(i, j int) bool {
+	favorWithValue := func(a, b string) bool {
+		if a == "" && b == "" {
+			return false
+		}
```

Comment:
nit: line break

Reply:
I added a line break here.

Do you have some rules I can align to, about when and where you would like to have line breaks?

My feeling says: either I omit the line breaks, since this is a very short function, where all the pieces belong together (no real paragraphs in the sense that they touch different things) or I would add an other new line also after the second `if` block (before the `return`).

Reply:
Essentially line break after every block is the coding style we've been going with.